packages/theme-language-server-node/src/dependencies.ts

```diff
@@ -4,8 +4,7 @@ import { basename } from 'node:path';
 import * as fs from 'node:fs/promises';
 import { promisify } from 'node:util';
 import { glob as callbackGlob } from 'glob';
-import { Config, Translations } from '@shopify/theme-check-common';
-import { loadConfig as loadConfigFromPath } from '@shopify/theme-check-node';
+import { Config, Translations, loadConfig as loadConfigFromPath } from '@shopify/theme-check-node';
```

Comment:
Because `@shopify/theme-check-common` wasn't a project reference of theme-language-server-node, we were loading the `dist` files here instead of the build ones. Meaning a `--watch` build would be using stale code if we were modifying any of those interfaces.